defaults/config.js

```diff
@@ -281,5 +281,15 @@ module.exports = {
 		// @default  113
 		//
 		port: 113
-	}
+	},
+
+	//
+	// Enable oidentd support using the specified file
+	//
+	// Example: oidentd: (process.env.HOME || process.env.USERPROFILE) + "/.oidentd.conf"
```

Comment:
Yes, unfortunately. I don't know if there's a way to have it expand that properly =/